projects/pastanaga-angular/src/lib/button/button.component.ts

```diff
@@ -46,27 +46,27 @@ export class ButtonComponent implements AfterContentInit {
             this._type = value;
         }
     }
-    @Input() set disabled(value: boolean) {
+    @Input() set disabled(value: any) {
         this._disabled = coerceBooleanProperty(value);
     }
-    @Input() set active(value: boolean) {
+    @Input() set active(value: any) {
         this._active = coerceBooleanProperty(value);
     }
     @Input() set icon(value: string) {
         this._icon = value || '';
     }
-    @Input() set iconAndText(value: boolean) {
+    @Input() set iconAndText(value: any) {
         this._iconAndText = coerceBooleanProperty(value);
     }
 
     @ViewChild('textContainer') textContainer?: ElementRef;
 
     _type: 'button' | 'submit' | 'reset' = 'button';
-    _kind: Kind = Kind.secondary;
-    _size: Size = Size.medium;
-    _aspect: Aspect = Aspect.solid;
+    _kind: Kind = 'secondary';
```

Comment:
It seems wrong. But visibly, my IDE is able to rename the symbol, so even if it looks like a string, it is actually a symbol according the compiler.
So why not.

Reply:
Kind, Aspect and Size are now union types like `Kind = 'primary' | 'secondary' | 'destructive';`.